tools/Repo-Tasks.psm1

```diff
@@ -293,3 +296,4 @@ export-modulemember -Function Get-BuildScopes
 export-modulemember -Function Start-Build
 export-modulemember -Function Invoke-CheckinTests
 export-modulemember -Function Install-VSProjectTemplates
+export-modulemember -Function Set-TestMode
```

Comment:
Not sure why this is there. I don't remember where I was going with it. Will be removed.